ckcp/hack/util/util-development-patch.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
```

Comment:
I'm not sure why it needs to be in another file.

Reply:
Yes, it doesn't need to be.